src/js/actions/GuideActions.js

```diff
@@ -1,24 +1,29 @@
-import AppDispatcher from "../dispatcher/AppDispatcher";
-import GuideConstants from "../constants/GuideConstants";
-
-const GuideActions = {
-  /**
-   * @param {String} id to ignore
-   */
-  ignore: function (id) {
-    AppDispatcher.dispatch({
-      actionType: GuideConstants.ORG_IGNORE, id
-    });
-  },
-
-  /**
-   * @param {String} id to follow
-   */
-  follow: function (id) {
-    AppDispatcher.dispatch({
-      actionType: GuideConstants.ORG_FOLLOW, id
-    });
-  }
-};
-
-export default GuideActions;
+var Dispatcher = require("../dispatcher/Dispatcher");
+
+var GuideActions = (function (_Dispatcher) {
+  function _GuideActions () { }
```

Comment:
One thing is that if this is already being `new`'ed up around the app, I would not expect you to change just for a personal preference that I have. 

The comments I have given are from those personal preferences and commenting on the whole file change example, I do not have a strong reason against the IIFE.